.drone.star

```diff
@@ -622,26 +622,39 @@ def website(ctx):
     },
     'steps': [
       {
-        'name': 'generate',
-        'image': 'webhippie/hugo:latest',
-        'pull': 'always',
+        'name': 'prepare',
+        'image': 'owncloud/alpine:latest',
         'commands': [
-          'make docs',
+          'apk add make',
```

Comment:
You should simply use a docker image that already includes make, why installing it on every pipeline run?

Reply:
suggesttions?

Reply:
I'll build one in owncloudci org....